neighbors_testcases_test.go

```diff
@@ -0,0 +1,514 @@
+package geohash
+
+//	# Test cases generated with ngeohash - Note that I had to remove some of the
```

Comment:
Indent? As mentioned in other comment, perhaps have your node code write its own source code into the output file?